cmd/influx/org_members.go

```diff
@@ -45,6 +45,11 @@ func newOrgMembersAddCmd() cli.Command {
 				EnvVar:      "INFLUX_ORG_ID",
 				Destination: &params.OrgID,
 			},
+			&cli.BoolFlag{
+				Name:        "owner",
```

Comment:
maybe allow `o` here as well for convenience?

Reply:
I wondered if users might accidentally use `-o` thinking they were passing an org flag, but actually set the user as owner. What do you think @samhld ?